### PR TITLE
Cleanup backend_bases docs.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/backend_bases.py [hunk 1]
```diff
@@ -1728,10 +1709,10 @@ def scroll_event(self, x, y, step, guiEvent=None):
 
         Backend derived classes should call this function on any
         scroll wheel event.  (*x*, *y*) are the canvas coords ((0, 0) is lower
-        left).  button and key are as defined in MouseEvent.
+        left).  button and key are as defined in `MouseEvent`.
 
-        This method will be call all functions connected to the
-        'scroll_event' with a :class:`MouseEvent` instance.
+        This method will be called all functions connected to the
```
Comment (timhoffm) [inline lib/matplotlib/backend_bases.py]:
```suggestion
        This method will call all functions connected to the
```

File: lib/matplotlib/backend_bases.py [hunk 2]
```diff
@@ -1437,21 +1422,17 @@ class PickEvent(Event):
...
     artist : `matplotlib.artist.Artist`
-        the picked artist
-
+        The picked artist.
     other
-        extra class dependent attrs -- e.g., a
-        :class:`~matplotlib.lines.Line2D` pick may define different
-        extra attributes than a
-        :class:`~matplotlib.collections.PatchCollection` pick event
+        Extra class dependent attrs -- e.g., a `~.Line2D` pick may define
```
Comment (timhoffm) [inline lib/matplotlib/backend_bases.py]:
~~~
Additional attributes may be present depending on the type of the
picked object; e.g. a `~.Line2D` pick may provide other additional
information than a `~PatchCollection` pick.
~~~

File: lib/matplotlib/backend_bases.py [hunk 3]
```diff
@@ -1897,7 +1878,7 @@ def release_mouse(self, ax):
             self.mouse_grabber = None
 
     def draw(self, *args, **kwargs):
-        """Render the :class:`~matplotlib.figure.Figure`."""
+        """Render the `~.Figure`."""
```
Comment (timhoffm) [inline lib/matplotlib/backend_bases.py]:
```suggestion
        """Render the `.Figure`."""
```

File: lib/matplotlib/backend_bases.py [hunk 4]
```diff
@@ -1748,10 +1729,10 @@ def button_press_event(self, x, y, button, dblclick=False, guiEvent=None):
 
         Backend derived classes should call this function on any mouse
         button press.  (*x*, *y*) are the canvas coords ((0, 0) is lower left).
-        button and key are as defined in :class:`MouseEvent`.
+        button and key are as defined in `MouseEvent`.
 
-        This method will be call all functions connected to the
-        'button_press_event' with a :class:`MouseEvent` instance.
+        This method will be called all functions connected to the
```
Comment (timhoffm) [inline lib/matplotlib/backend_bases.py]:
```suggestion
        This method will call all functions connected to the
```

File: lib/matplotlib/backend_bases.py [hunk 5]
```diff
@@ -598,7 +594,7 @@ def get_text_width_height_descent(self, s, prop, ismath):
         """
         Get the width, height, and descent (offset from the bottom
         to the baseline), in display coords, of the string *s* with
-        :class:`~matplotlib.font_manager.FontProperties` *prop*
+        `.FontProperties` *prop*
```
Comment (timhoffm) [inline lib/matplotlib/backend_bases.py]:
```suggestion
        `.FontProperties` *prop*.
```

**Comment by anntzer** [comment]
thanks, all handled

**Pull request merged**